xep-0353.xml

```diff
@@ -270,7 +276,7 @@
   <p>Thanks to Lance Stout for his feedback.</p>
 </section1>
 <section1 topic='Security Considerations' anchor='security'>
-  <p>Because exchanging messages with other entities is effectively is a presence leak, an XMPP client that implements the receiving side of this specification MUST disable sending of proceed messages by default and MUST enable the feature only as a result of explicit user confirmation. Such confirmation can be provided per request, by whitelisting requests received from Jingle initiators in the responder's contact list, or through some other suitable means as long as sending proceed messages does not occur by default.</p>
+  <p>Because exchanging messages with other entities is effectively is a presence leak, an XMPP client that implements the receiving side of this specification MUST disable sending of proceed messages by default and MUST enable the feature only as a result of explicit user confirmation. Such confirmation can be provided per request, by only allowing requests received from Jingle initiators in the responder's contact list, or through some other suitable means as long as sending proceed messages does not occur by default.</p>
```

Comment:
I think this subtly changes the meaning - to me the whitelist text implied that it was "automatically allowing" rather than "only allowing".

Reply:
Good question. I had originally written simply "allowing", but actually got the opposite impression, and that "whitelist" was equivalent to "allow only" rather than "[may] allow".

On re-reading it you're probably right, it was fine. The sentence as a whole is pretty clear of its intention either way. I've switched "only" to "automatically".